components/klaudsolcms/AppModal.js

```diff
@@ -1,7 +1,8 @@
 import Modal from 'react-bootstrap/Modal';
 import Button from 'react-bootstrap/Button';
 import { FaTimes } from 'react-icons/fa'
-const AppModal = ({show, onClose, onClick, modalTitle, children, buttonTitle}) => {
+import AppButtonSpinner from '@/components/klaudsolcms/AppButtonSpinner';
+const AppModal = ({show, onClose, onClick, modalTitle, children, buttonTitle,isLoading}) => {
```

Comment:
If there are no statements between const AppModal = ( { show, onClose, onClick, modalTitle, children, buttonTitle,isLoading } ) => { (curly braces) and return (, you can just omit return ( and use const AppModal = ( { show, onClose, onClick, modalTitle, children, buttonTitle, isLoading } ) => ( (parentheses) instead.

Additionally, use the correct application of new lines for code readability like so:

```suggestion
const AppModal = ({ 
  show, 
  onClose, 
  onClick, 
  modalTitle, 
  children, 
  buttonTitle,
  isLoading
  }) => (
     <Modal 
       show={show} 
       onHide={onClose} 
       centered 
       size="lg"
     >
  );
```

Reply:
Noted ma'am. Thankyou.